k8s/dns-ssl-kube-lego/README.md

```diff
@@ -0,0 +1,198 @@
+# Endpoints DNS with SSL certificate issued by Let's Encrypt through Kube-Lego
+
+This example demonstrates how to deploy an Endpoints service on GKE with automatic
+provisioning of an SSL certificate for the .cloud.goog service domain.
+
+## Requirements
+
+For this example, you need:
+
+ -  A proper installation of the Cloud SDK, configured with an account and a project:
+    ```
+        $ gcloud config list
+        [core]
+        account = ...
+        project = ...
+    ```
+    Make sure your credentials are up-to-date:
+    ```
+        $ gcloud auth login
+    ```
+- A proper GKE cluster created in your project:
+    ```
+        $ CLUSTER_NAME=endpoints-dns-sample
+        $ gcloud container clusters create $CLUSTER_NAME --zone=us-west1-a --num-nodes=3
+        $ gcloud container clusters get-credentials $CLUSTER_NAME
+    ```
+    Notes: Creating a new GKE cluster make take several minutes.
+
+## Deployment
+
+You can deploy the example by running the `deploy.sh` script.
+The script will confirm your account email and your GCP project ID.
+
+### Detailed steps
```

Comment:
It's unclear whether this section explains what happens when you run `deploy.sh`, or whether you need to follow it after running `deploy.sh`. I'd add a note that you can either use the shell script or follow the detailed steps, not both.

Reply:
Updated paragraph accordingly.